src/windows/webview.c

```diff
@@ -477,21 +478,23 @@ static ICoreWebView2EnvironmentOptionsVtbl VTBL5 = {
 
 // Public
 
-static HMODULE webview_load_dll(void)
+static bool webview_dll_path(WCHAR *pathw, bool as_user)
 {
+	bool ok = false;
+
 	HKEY key = NULL;
-	WCHAR *dll = NULL;
-	HMODULE lib = NULL;
+	WCHAR dll[MTY_PATH_MAX] = {0};
+
+	// https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#detect-if-a-webview2-runtime-is-already-installed
+	const wchar_t *machine_path = L"Software\\Microsoft\\EdgeUpdate\\ClientState\\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}";
+	const wchar_t *user_path = L"Software\\Microsoft\\EdgeUpdate\\Clients\\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}";
+	const wchar_t *reg_path = as_user ? user_path : machine_path;
 
-	LSTATUS r = RegOpenKeyEx(HKEY_LOCAL_MACHINE,
-		L"Software\\Microsoft\\EdgeUpdate\\ClientState\\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}",
-		0, KEY_WOW64_32KEY | KEY_READ, &key);
+	LSTATUS r = RegOpenKeyEx(HKEY_LOCAL_MACHINE, reg_path, 0, KEY_WOW64_32KEY | KEY_READ, &key);
 	if (r != ERROR_SUCCESS)
 		goto except;
 
-	dll = MTY_Alloc(MAX_PATH, sizeof(WCHAR));
-	DWORD size = MAX_PATH * sizeof(WCHAR);
```

Comment:
Why are you changing `MAX_PATH` to `MTY_PATH_MAX`? They are not the same, and you are never using `dll` in MTY functions...it is only used in windows API funcs, which expect `MAX_PATH` to be honoured, I thought.

Reply:
Oops, I'll fix that